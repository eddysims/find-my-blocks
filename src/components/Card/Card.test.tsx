import React from "react";
import { cleanup, render } from "@testing-library/react";
import { Card } from ".";

afterEach(cleanup);

it("renders correctly", () => {
  const { container } = render(<Card title="Toronto">Go Raps!</Card>);
  expect(container).toMatchSnapshot();
});

it("renders without a title", () => {
  const { container } = render(<Card>Go Raps!</Card>);
  expect(container).toMatchSnapshot();
});
