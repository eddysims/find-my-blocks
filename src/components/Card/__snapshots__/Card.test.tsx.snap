// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    class="components-card is-size-small css-1xs3c37-CardUI e1q7k77g0"
  >
    <div
      class="components-card__header is-shady is-size-small css-1vhqpef-HeaderUI e1q7k77g1"
    >
      <strong>
        Toronto
      </strong>
    </div>
    <div
      class="components-card__body is-size-small css-xmjzce-BodyUI e1q7k77g3"
    >
      Go Raps!
    </div>
  </div>
</div>
`;

exports[`renders without a title 1`] = `
<div>
  <div
    class="components-card is-size-small css-1xs3c37-CardUI e1q7k77g0"
  >
    <div
      class="components-card__body is-size-small css-xmjzce-BodyUI e1q7k77g3"
    >
      Go Raps!
    </div>
  </div>
</div>
`;
